src/preview.js

```diff
@@ -76,19 +75,15 @@ const renderPreview = ( lang, data, isTouch ) => {
 	renderDisambiguation = ( isTouch, lang, title, dir ) => {
 		return `
 			<div class="wikipediapreview ${isTouch ? 'mobile' : ''}" lang="${lang}" dir="${dir}">
-					<div class="wikipediapreview-header">
-						<div class="wikipediapreview-loading-header-image"></div>
-						<div class="wikipediapreview-header-wordmark wikipediapreview-header-wordmark-${lang}"></div>
-						<div class="wikipediapreview-header-closebtn"></div>
-					</div>
+					${getPreviewHeader( lang )}
 					<div class="wikipediapreview-disambiguation">
 						<div class="wikipediapreview-disambiguation-body">
 							<div class="wikipediapreview-disambiguation-body-message">
 								<div class="wikipediapreview-disambiguation-body-icon"></div>
 								<span>${msg( lang, 'preview-disambiguation-message', title )}</span>
 							</div>
 							<div class="wikipediapreview-disambiguation-body-readon">
-								<a href=${`https://${lang}.m.wikipedia.org/wiki/${encodeURIComponent( title )}`} target="_blank">${msg( lang, 'read-on-wiki' )}</a>
+								<a href=${`https://${lang}.m.wikipedia.org/wiki/${encodeURIComponent( title )}`} target="_blank">${msg( lang, 'read-on-wiki-disambiguation' )}</a>
```

Comment:
We decided not to introduce a distinct CTA here when we did the disambiguation task. I'm guessing the mockups were never updated so you saw the discrepancy.